.gitignore

```
@@ -7,3 +7,5 @@
 /dist/
 __pycache__/
 build
+.idea/
+*venv/
```

Can you make sure there's a new line at the end of this file? That way `echo` on it works.

```suggestion
*venv/

```

---

Also optionally, if you don't mind getting close and personal with git rebasing, it would be nice if this was a separate commit (so I can rebase two on top of master.)

---

I don't think I understood what you meant with rebasing. I just commited the two changes to the PR. Hope that is okay.